packages/liveblocks-client/src/lson.ts

```diff
@@ -0,0 +1,73 @@
+import type { LiveList } from "./LiveList";
+import type { LiveMap } from "./LiveMap";
+import type { LiveObject } from "./LiveObject";
+import type { LiveRegister } from "./LiveRegister";
+import type { Json } from "./json";
+
+/**
+ * Think of Lson as a sibling of the Json data tree, except that the nested
+ * data structure can contain a mix of Json values and LiveXxx instances.
+ */
+export type Lson =
```

Comment:
I love that! 😍 

Easy to understand and catchy name! So useful to make sure no one is passing a function or other non serializable values! There is probably a way to also use it on zustand mapping to make sure no one is trying to synchronize a function on the store.

Reply:
> There is probably a way to also use it on zustand mapping to make sure no one is trying to synchronize a function on the store.

Can figure out, will be nice to include for sure!